apps/web/src/layouts/DefaultLayout/NavMenu.tsx

```diff
@@ -264,7 +261,9 @@ export const NavMenu = () => {
                     {viewableDaos?.map((dao, index) => (
                       <Link
                         key={dao.collectionAddress}
-                        href={`/dao/${selectedChain.slug}/${dao.collectionAddress}`}
+                        href={`/dao/${
+                          PUBLIC_DEFAULT_CHAINS.find((x) => x.id === dao.chainId)?.slug
```

Comment:
This will run every time the component renders. I know the range is constrained (# of chains) but I would still suggest memoizing.

Reply:
I think memoizing would cost more than keeping this computation as is. the operation is pretty inexpensive since we are only iterating on 3 chains.

https://blog.logrocket.com/when-not-to-use-usememo-react-hook/